share/completions/complete.fish

```diff
@@ -15,6 +16,7 @@ complete -c complete -s h -l help -d "Display help and exit"
 complete -c complete -s C -l do-complete -d "Print completions for a commandline specified as a parameter"
 complete -c complete -s n -l condition -d "Completion only used if command has zero exit status" -x
 complete -c complete -s w -l wraps -d "Inherit completions from specified command" -xa '(__fish_complete_command)'
+complete -c complete -s k -l keep-order -d "Keep order of option arguments instead of sorting alphabetically"
```

Comment:
I suggest "Keep ordering of arguments instead of sorting alphabetically"

Reply:
"arguments" seems to broad, "option arguments" is more clear and already used in the description of `-a/--arguments`